protocol/dubbo3/dubbo3_invoker.go

```diff
@@ -59,8 +59,8 @@ func NewDubboInvoker(url *common.URL) (*DubboInvoker, error) {
 	rt := config.GetConsumerConfig().RequestTimeout
 
 	timeout := url.GetParamDuration(constant.TIMEOUT_KEY, rt)
-	key := url.GetParam(constant.BEAN_NAME_KEY, "")
```

Comment:
This is a bug, for triple pb scene. The bean.name can't locate the target consumer stub , so we use interface.